frontends/android/BitBoxApp/app/src/main/AndroidManifest.xml

```diff
@@ -34,6 +34,9 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/AppTheme">
+        <!-- This tag might be required when interacting with Google Play Services, see https://developers.google.com/android/guides/setup. -->
```

Comment:
my suggestion was `This tag is needed` not `This tag might be required`, as I had the impression it was needed. Not sure why you changed it :) anyway